docs/pages/dsFPD/ds-front-panel-display_L3_Low-Level_TestSpecification.md

```diff
@@ -96,4 +96,4 @@ classDiagram
 
 - **testSuite.yaml**
```

Comment:
Change Name here

Reply:
updated name from testSuite to testConfig.